<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LehelugejaAken</class>
 <widget class="QDialog" name="LehelugejaAken">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1026</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Protokollitaja - Lehelugeja</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>598</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="silt">
         <property name="minimumSize">
          <size>
           <width>600</width>
           <height>600</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>600</width>
           <height>600</height>
          </size>
         </property>
         <property name="text">
          <string>Siia tuleb märkleht :)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0" colspan="2">
        <widget class="QTextEdit" name="logi">
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="tekstiEdit"/>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="saadaNupp">
         <property name="text">
          <string>Saada</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Esimese lehe siffer:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="nimeSilt">
         <property name="text">
          <string>Nimi:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3" colspan="2">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Seeriad:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="9">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Summa:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLineEdit" name="sifriEdit">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="nimeBox">
         <property name="minimumSize">
          <size>
           <width>125</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QToolButton" name="peidaNupp">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>&gt;&gt;</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QLineEdit" name="seeria1Edit">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QLineEdit" name="seeria2Edit">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="5">
        <widget class="QLineEdit" name="seeria3Edit">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="6">
        <widget class="QLineEdit" name="seeria4Edit">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="7">
        <widget class="QLineEdit" name="seeria5Edit">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="8">
        <widget class="QLineEdit" name="seeria6Edit">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="9">
        <widget class="QLineEdit" name="summaEdit">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="sifriLabel">
         <property name="text">
          <string>Siffer</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5" rowspan="2" colspan="2">
        <widget class="QPushButton" name="resetNupp">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Alusta uuesti</string>
         </property>
        </widget>
       </item>
       <item row="0" column="7" rowspan="2" colspan="2">
        <widget class="QPushButton" name="liigneNupp">
         <property name="text">
          <string>Liigne lask</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="2">
        <widget class="QPushButton" name="sulgeUhendusNupp">
         <property name="text">
          <string>Sulge ühendus</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Lehtede liik:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="laskudeBox">
         <property name="readOnly">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Laskude arv lehes:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QToolButton" name="peida2Nupp">
         <property name="toolTip">
          <string>Näitab rohkem nuppe</string>
         </property>
         <property name="text">
          <string>&gt;&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QComboBox" name="comPort">
         <property name="editable">
          <bool>true</bool>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>COM1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM9</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM10</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="seadistaNupp">
         <property name="text">
          <string>Seadista</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="leheCombo">
         <item>
          <property name="text">
           <string>Õhupüss</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Õhupüstol</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sportpüss</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QCheckBox" name="kumnendikegaBox">
         <property name="text">
          <string>Loe kümnendikega</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="uhendaNupp">
         <property name="text">
          <string>Ühenda</string>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="otsiCOMNupp">
         <property name="text">
          <string>Otsi ja ühenda</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>947</width>
         <height>22</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="sulgeNupp">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Sulge</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>sulgeNupp</sender>
   <signal>clicked()</signal>
   <receiver>LehelugejaAken</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>968</x>
     <y>753</y>
    </hint>
    <hint type="destinationlabel">
     <x>944</x>
     <y>693</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
